File: aaah.cpp
// GitHub: EntityPlantt/Kattis
#include <iostream>
using namespace std;
int main() {
    string a, b;
    cin >> a >> b;
    if (a.size() < b.size()) {
        cout << "no";
    }
    else {
        cout << "go";
    }
    return 0;
}

File: aaah.js
// GitHub: EntityPlantt/Kattis
var l = null;
require("readline").createInterface({
	input: process.stdin,
	output: process.stdout
}).on("line", line => {
	if (l == null) {
		l = line.length;
	}
	else {
		console.log(l < line.length ? "no" : "go");
		process.exit();
	}
});